packages/frontend-near-wallet/src/utils/constants.ts

```diff
@@ -0,0 +1,21 @@
+import Environments from '../../../../features/environments.json';
+import { NEAR_WALLET_ENV } from './config';
+
+export const WALLET_MIGRATION_VIEWS = {
+    MIGRATION_PROMPT: 'MIGRATION_PROMPT',
+    SELECT_DESTINATION_WALLET: 'SELECT_DESTINATION_WALLET',
+    MIGRATION_SECRET: 'MIGRATION_SECRET',
+    MIGRATE_ACCOUNTS: 'MIGRATE_ACCOUNTS'
+
+};
+
+export const getMyNearWalletUrl = (https = true) => {
```

Comment:
We need to make sure `NEAR_WALLET_ENV` is set to non `*_NEARORG` when we deploy the new landing page or use `getMyNearWalletUrlFromNEARORG`. Pretty sure there are conflicts here since `encodeAccountsToURL` uses `getMyNearWalletUrlFromNEARORG` and `RedirectPage` uses `getMyNearWalletUrl`. Please test the whole thing with a set `NEAR_WALLET_ENV` and flag a deploy with that `NEAR_WALLET_ENV` when it's time to deploy

Reply:
This is scheduled to be handled in a follow up task. The flow for redirect will change based on the new design. We want to allow users select a destination wallet of their choice and they'd get redirected accordingly.

Reply:
Still not clear what you intend `NEAR_WALLET_ENV` to be set to when you deploy this. It is being used to compute the export url too